kitsune/wiki/views.py

```diff
@@ -542,6 +542,22 @@ def review_revision(request, document_slug, revision_id):
     revision_contributors = list(set(
         based_on_revs.values_list('creator__username', flat=True)))
 
+    # Get latest revision which is still not approved. Use **latest_revision_id** to get the id.
+    try:
+        latest_revision = Revision.objects.filter(
+            document=doc,
+            is_approved=False).order_by('-id')[0]
+        latest_revision_id = latest_revision.id
+    except IndexError:
+        latest_revision_id = None
```

Comment:
This variable is not well named. It is not simply the latest revision's id, but the latest *unapproved* revision. That should be reflected in the name.